inc/saivlan.h

```diff
@@ -266,14 +266,63 @@ typedef enum _sai_vlan_attr_t
     SAI_VLAN_ATTR_META_DATA,
 
     /**
-     * @brief To disable flooding traffic (Broadcast, unknown unicast,
-     * unknown multicast) on a VLAN
+     * @brief Unknown unicast flood group.
      *
-     * @type bool
+     * Provides control on the set of bridge ports on which unknown unicast
+     * packets need to be flooded.By default without setting this attribute,
+     * unknown unicast packets would be flooded to all vlan members. If this
+     * attribute is set with a L2MC group id, then unknown unicast packets
+     * would be flooded to the L2MC group members. If this attribute is set
+     * with NULL object id, the default behavior to flood unknown unicast
+     * packets to all vlan members would be restored.
+     *
+     * @type sai_object_id_t
      * @flags CREATE_AND_SET
-     * @default false
+     * @objects SAI_OBJECT_TYPE_L2MC_GROUP
+     * @allownull true
+     * @default SAI_NULL_OBJECT_ID
+     */
+    SAI_VLAN_ATTR_UNKNOWN_UNICAST_FLOOD_GROUP,
```

Comment:
need another enum attribute to define where to flood to all vlan member, none or use l2mc group.